Wrapping/Python/Examples/use_case_02.py

```diff
@@ -43,38 +44,44 @@ def import_csv_xyz_vertices_2(file_path: str):
     dc.setGeometry(vertGeom)
     # Add the data container to the DataContainerArray
     dca.addOrReplaceDataContainer(dc)
-    # Write the Vertex Geometry out to a .dream3d file.
-    err = simpl_helpers.WriteDREAM3DFile('vert_geom_2.dream3d', dca, True)
     
+    file_path=args.output_file
+    file_path: str
+    numVerts = len(verts_np)
+
+    #Creates csv with same amount of elements as vertices from input file
+    make_random_data(numVerts, file_path=args.output_file)
+
     # Create Cell AttributeMatrix
-    tupleDims = simpl.VectorSizeT([1, 37989])
-    am = simpl_helpers.CreateAttributeMatrix(tupleDims, 'CellAttributeMatrix', simpl.AttributeMatrix.Type.Cell)
+    tupleDims = simpl.VectorSizeT([1, numVerts])
+    am = simpl_helpers.CreateAttributeMatrix(tupleDims, 'CellAttributeMatrix', simpl.AttributeMatrix.Type.Vertex)
 
     # Create Data Arrays
     #Loads in csv file and gets data for each type
-    uints = np.genfromtxt('new_data.csv', dtype=np.uint32, skip_header=1, usecols=1, delimiter=',')
-    f32s = np.genfromtxt('new_data.csv', dtype=np.float32, usecols=0, skip_header=1, delimiter=',')
+    uints = np.genfromtxt(file_path, dtype=np.uint32, skip_header=1, usecols=1, delimiter=',')
+    f32s = np.genfromtxt(file_path, dtype=np.float32, usecols=0, skip_header=1, delimiter=',')
 
     #Creates arrays from data with shape of (# of vertices x 1)
-    uintarr = np.ndarray(shape=(37989, 1), dtype=np.uint32, buffer=uints)
-    floatarr = np.ndarray(shape=(37989, 1), dtype=np.float32, buffer=f32s)
+    uintarr = np.ndarray(shape=(numVerts, 1), dtype=np.uint32, buffer=uints)
+    floatarr = np.ndarray(shape=(numVerts, 1), dtype=np.float32, buffer=f32s)
+
+    #Makes simpl arrays using numpy arrays
+    uint32arr = simpl.UInt32Array(uintarr, 'UInt32 Arr', True)
+    f32arr = simpl.FloatArray(floatarr, 'Float32 Arr', True)
 
     #     Number of tuples must match the number of vertices
     #     Create 2 Data Arrays: UInt32 and Float32
     #     Populate data ararys with some random data
     #       or
     #     for bonus points read the data from another text file
-    #Makes simpl arrays using numpy arrays
-    uint32arr = simpl.UInt32Array(uintarr, 'UInt32 Arr', True)
-    f32arr = simpl.FloatArray(floatarr, 'Float32 Arr', True)
-    
     # Add Data Arrays to AttributeMatrix
     am.addOrReplaceAttributeArray(uint32arr)
     am.addOrReplaceAttributeArray(f32arr)
 
     # Add AttributeMatirx to DataContainer
     dc.addOrReplaceAttributeMatrix(am)
 
+    err = simpl_helpers.WriteDREAM3DFile('vert_geom_2.dream3d', dca, True)
```

Comment:
The output file path should be a concatenation of the output_dir variable the actual name of the output file, in this instance 'vert_geom_2.dream3d'.

Please update all places in the code where output files are written.